### PR TITLE
Create ken111989.json

### PR DESCRIPTION
**Pull request by ken111989**
Adding Columbia to the map!

### CODE REVIEW

File: _pins/ken111989.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+gitbubhandle: ken111989
```
Comment (brianamarie) [inline _pins/ken111989.json]: Hi @ken111989! Make sure the spelling here is correct, `githubHandle`, so the tests will pass. Let me know if you want any help!

File: _pins/ken111989.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+latitude: 34.167137
+longitude: -80.945433
```
Comment (brianamarie) [inline _pins/ken111989.json]: @ken111989 You'll also want to add three lines to the last line, like `---`.

**Comment by debrajoseph** [comment]
@brianamarie Just practising

**Comment by crichID** [comment]
@ken111989 your code looks perfect! Please go ahead and click the big green button below to **Merge pull request**. In an effort to keep things in this repository clean, I will merge this if it is still open on 3/17/2017.

**Pull request merged**